cmd/esc/cli/env_version_retract.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023, Pulumi Corporation.
+
+package cli
+
+import (
+	"context"
+	"errors"
+
+	"github.com/spf13/cobra"
+)
+
+func newEnvVersionRetractCmd(env *envCommand) *cobra.Command {
+	var replacement string
+	var reason string
+
+	cmd := &cobra.Command{
+		Use:   "retract [<org-name>/]<environment-name>@<version>",
```

Comment:
one question--should this be a peer of `rollback`, which is currently `esc env rollback`? or should we move `rollback` under `version`?

Reply:
I do think we should be consistent, I think both should be under version

Reply:
Okay. I'll follow up with a PR to move `rollback` under `version`.